index.bs

```diff
@@ -2293,35 +2293,72 @@ The W3C Council</h4>
+	<ol>
+		<li>
+			They filed the formal objection, helped author it, 
+			or work for the same organization as the author/filer
+		<li>
+			They made the decision being objected to
+		<li>
+			They were an editor of, or a major contributor to, the document in question
+		<li>
+			Their business depends on the question (in either direction)
+		<li>
+			They hold sufficiently strong pre-existing opinions in either direction 
+			(e.g. a well-known proponent or objector) that their ability 
+			to be open-minded may reasonably be in doubt
+		<li>	
+			They themselves offered the reason and asked that it be stated
+		<li>
+			A member of the W3C membership or team feels that they 
+			should be dismissed and has presented a reason
+	</ol>
+
+	Before the Council forms, the team assignee presents the entire list of potential members and collected reasons 
+	to the potential council; 
+	the potential council then votes whether to dismiss each potential member, 
+	considering whether the reasons given indicate that that individual's participation 
+	would compromise the integrity of the Council decision.
+	No potential member is allowed to vote on their own dismissal; 
+	each dismissal is decided by simple majority of those voting yes/no (abstentions on a question of dismissal not affecting the result).
+				
+	[=Dismissal=] is individual;
+	when the decision in question was made by the [=TAG=] or [=AB=] acting as a body,
+	the entire [=TAG=] or [=AB=] is not expected or required to be dismissed.
+
+	An individual <em class=rfc2119>may</em> also [=renounce=] their seat on the Council, for strong reason,
+	such as being forbidden by their employer to serve. The individual chooses the extent to which they explain
+	their renunciation.
+	
+	However, renunciation is disqualification from participation,
 	not abstention,
 	and <em class=rfc2119>should not</em> be used
 	to excuse an absence of participation.
-	Any person who has <dfn lt="recusal | recuse | recuses | recused">recused</dfn> themself
-	does not take part in its deliberations,
-	determination of consensus,
-	or votes
-	on this matter.
+	
+	Any member may abstain from attending or forming part of the consensus body.
+				
+	Any person who has been <dfn lt="dismissed | dismiss | dismissal">dismissed</dfn> 
+	or who <dfn lt="renounce | renunciation | renounces">renounces</dfn> their seat
+	does not receive materials, take part in its deliberations, help in the determination of consensus,
+	or partcipate in votes on this matter.
 	The [=W3C Council=] <em class=rfc2110>may</em> still solicit and hear their testimony,
 	as they can of anyone else in the W3C community.
-	[=Recusal=] is individual;
-	when the decision in question was made by the [=TAG=] or [=AB=] acting as a body,
-	the entire [=TAG=] or [=AB=] is not expected or required to recuse itself.
-
+	
+	The Council report includes a report on the names of those who were dismissed, renounced their seat, or
+	were qualified to serve.
```

Comment:
Should the report also include (either in positive or negative form) the list of those who attended the council meetings / participated in the deliberations (vs those who skipped)?

Reply:
I suspect a Council member can ask to be reported as having abstained, and that "not present for the decision" should be reported as well, yes.

Dismissed: Mordred, Amalasuintha
Potentially serving but voluntarily abstaining: Gaiseric, Merlin
Potentially serving but not present: Galla Placidia
Serving: Tamurlane, Pippin the Hunchback, Karl the Younger, Louis the Pious, Engelberga, Marozia of Tusculum